src/main/java/edu/unc/lib/boxc/migration/cdm/services/PermissionsService.java

```diff
@@ -216,6 +245,61 @@ private List<String> queryForMappedIds(PermissionMappingOptions options) {
         return mappedIds;
     }
 
+    private List<List<String>> updateCsvRecords(PermissionMappingOptions options, Path permissionsMappingPath) throws Exception {
+        List<List<String>> updatedRecords = new ArrayList<>();
+        List<String> cdmIds = new ArrayList<>();
```

Comment:
It won't make too big a difference for the scale of records here since it's only checking for one id, but often a `Set`/`HashSet` is a more appropriate data structure in cases where we are collecting ids for lookups or deduplication, since order doesn't matter and `contains` calls are much faster.